### PR TITLE
feat(Homepage): add Chinese translation

### PR DESCRIPTION
**Pull request by ChungZH**

### CODE REVIEW

File: content/zh/_index.html [hunk 1]
```diff
@@ -20,10 +20,10 @@
 			<div class="row">
 				<div class="col-lg-5">
 					<div class="banner_content">
-						<h2>The Missing Editor <br />for Competitive Programmers</h2>
-						<p>CP Editor is specially designed for competitive programming. It helps you focus on your algorithms and automates compilation, execution and testing of your code. It even fetches test cases for you from different platforms and submits solutions to Codeforces!</p>
-						<a class="banner_btn" href="https://github.com/cpeditor/cpeditor">View source</a>
-						<a class="banner_btn2" id="download_by_os" href="https://github.com/cpeditor/cpeditor/releases/latest">Download</a>
+						<h2>缺失的编辑器 <br />为算法竞赛程序员而生</h2>
```
Comment (ouuan) [inline content/zh/_index.html:23]:
```suggestion
						<h2>算法竞赛选手们<br />所需要的 IDE</h2>
```

File: content/zh/_index.html [hunk 2]
```diff
@@ -20,10 +20,10 @@
...
+						<p>CP Editor 专为算法竞赛设计，不像其它主要是为了开发设计的 IDE。它可以帮助你自动化编译、运行、测试，从而让你专注于算法设计。它甚至可以从各种算法竞赛网站上获取样例，将代码提交到 Codeforces 上！</p>
```
Comment (ouuan) [inline content/zh/_index.html:24]:
```suggestion
						<p>CP Editor 专为算法竞赛设计，不像其它 IDE 主要是为了开发设计的。它可以帮助你自动化编译、运行、测试，从而让你专注于算法设计。它甚至可以从各种算法竞赛网站上获取样例，将代码提交到 Codeforces 上！</p>
```

File: content/zh/_index.html [hunk 3]
```diff
@@ -41,29 +41,29 @@ <h2>The Missing Editor <br />for Competitive Programmers</h2>
 <div class="work_area p_120">
 	<div class="container">
 		<div class="main_title">
-			<h2>How it works for you</h2>
-			<p>CP Editor eliminates all the repetitive and boring tasks during algorithmic contests. It automates all that's reasonably possible and allows you to purely focus on your code.</p>
+			<h2>它如何为你工作</h2>
+			<p>CP Editor 消除了算法竞赛中的所有重复和无聊的任务。它会自动执行所有可能的操作，并让你完全专注于代码。</p>
```
Comment (ouuan) [inline content/zh/_index.html:45]:
```suggestion
			<p>CP Editor 解决了算法竞赛中重复和无聊的任务。它会尽可能地将操作自动化，让你完全专注于代码。</p>
```

File: content/zh/_index.html [hunk 4]
```diff
@@ -41,29 +41,29 @@ <h2>The Missing Editor <br />for Competitive Programmers</h2>
...
 		</div>
 		<div class="work_inner row">
 			<div class="col-lg-4">
 				<div class="work_item">
 					<i class="lnr lnr-screen"></i>
-					<a><h4>Fetch testcases</h4></a>
-					<p>CP Editor can fetch testcases automatically from almost all competitive programming sites. Say bye to error-prone copy-pasting.</p>
+					<a><h4>拉取测试用例</h4></a>
```
Comment (ouuan) [inline content/zh/_index.html:51]:
```suggestion
					<a><h4>获取测试用例</h4></a>
```

File: content/zh/_index.html [hunk 5]
```diff
@@ -41,29 +41,29 @@ <h2>The Missing Editor <br />for Competitive Programmers</h2>
...
+					<p>CP Editor 可以从几乎所有的算法编程站点中自动获取测试用例。告别容易出错的复制粘贴。</p>
```
Comment (ouuan) [inline content/zh/_index.html:52]:
```suggestion
					<p>CP Editor 可以从几乎所有的算法竞赛网站上自动获取样例。你再也不用手动进行麻烦的复制粘贴。</p>
```

File: content/zh/_index.html [hunk 6]
```diff
@@ -41,29 +41,29 @@ <h2>The Missing Editor <br />for Competitive Programmers</h2>
...
 				</div>
 			</div>
 			<div class="col-lg-4">
 				<div class="work_item">
 					<i class="lnr lnr-code"></i>
-					<a><h4>Code faster</h4></a>
-					<p>Code faster with its amazing customizable code snippets. In just two seconds you can insert a whole DFS code into your editor.</p>
+					<a><h4>更快地编程</h4></a>
+					<p>令人惊叹的自定义代码段可加快编程速度。在短短两秒钟内，你可以将整个 DFS 代码插入编辑器。</p>
```
Comment (ouuan) [inline content/zh/_index.html:59]:
```suggestion
					<p>强大的可自定义代码片段功能可加快编程速度。在短短两秒钟内，你可以将整个 DFS 代码插入编辑器。</p>
```

File: content/zh/_index.html [hunk 7]
```diff
@@ -41,29 +41,29 @@ <h2>The Missing Editor <br />for Competitive Programmers</h2>
...
 				</div>
 			</div>
 			<div class="col-lg-4">
 				<div class="work_item">
 					<i class="lnr lnr-clock"></i>
-					<a href="#"><h4>Get verdicts</h4></a>
-					<p>You can get verdicts for your sample testcases in one click by built-in checkers or checkers from testlib. You can also add your own checkers.</p>
+					<a href="#"><h4>获取判决</h4></a>
```
Comment (ouuan) [inline content/zh/_index.html:65]:
```suggestion
					<a href="#"><h4>获取评测结果</h4></a>
```

File: content/zh/_index.html [hunk 8]
```diff
@@ -41,29 +41,29 @@ <h2>The Missing Editor <br />for Competitive Programmers</h2>
...
+					<p>你可以通过内置检查器或来自 testlib 的检查器一键获取示例测试用例的判决。你也可以添加自己的检查器。</p>
```
Comment (ouuan) [inline content/zh/_index.html:66]:
```suggestion
					<p>你可以通过内置检查器或来自 testlib 的检查器一键获取样例的评测结果。你也可以添加自己的 SPJ。</p>
```

File: content/zh/_index.html [hunk 9]
```diff
@@ -77,27 +77,27 @@ <h2>How it works for you</h2>
 		<div class="made_life_inner">
 			<ul class="nav nav-tabs" id="myTab" role="tablist">
 				<li class="nav-item">
-				<a class="nav-link active" id="home-tab" data-toggle="tab" href="#withoutSaving" role="tab" aria-controls="home" aria-selected="true">Run without Saving</a>
+				<a class="nav-link active" id="home-tab" data-toggle="tab" href="#withoutSaving" role="tab" aria-controls="home" aria-selected="true">运行而不保存</a>
```
Comment (ouuan) [inline content/zh/_index.html:80]:
```suggestion
				<a class="nav-link active" id="home-tab" data-toggle="tab" href="#withoutSaving" role="tab" aria-controls="home" aria-selected="true">运行而无需保存</a>
```

File: content/zh/_index.html [hunk 10]
```diff
@@ -77,27 +77,27 @@ <h2>How it works for you</h2>
...
 			<div class="tab-content" id="myTabContent">
 				<div class="tab-pane fade show active" id="withoutSaving" role="tabpanel" aria-labelledby="home-tab">
 					<div class="row made_life_text">
 						<div class="col-lg-6">
 							<div class="left_side_text">
-								<h3>Run any code<br />At any time</h3>
-								<h6>We know that you want to test something without a temp file</h6>
-								<p>You can run your code in one click without saving and the hot exit feature makes it easy to restore your code from the last session.</p>
-								<a class="main_btn" href="#">Get Started Now</a>
+								<h3>运行任何代码<br />在任何时候</h3>
```
Comment (ouuan) [inline content/zh/_index.html:97]:
```suggestion
								<h3>在任何时候<br />运行任何代码</h3>
```

File: content/zh/_index.html [hunk 11]
```diff
@@ -77,27 +77,27 @@ <h2>How it works for you</h2>
...
+								<h6>我们知道你想要在没有临时文件的情况下进行测试</h6>
```
Comment (ouuan) [inline content/zh/_index.html:98]:
```suggestion
								<h6>我们知道你想要无需临时文件就能进行测试</h6>
```

File: content/zh/_index.html [hunk 12]
```diff
@@ -111,10 +111,10 @@ <h6>We know that you want to test something without a temp file</h6>
 					<div class="row made_life_text">
 						<div class="col-lg-6">
 							<div class="left_side_text">
-								<h3>Dark themes? <br />We've got you covered</h3>
-								<h6>Our lives are colorful, why not our coding editor</h6>
-								<p>We have added five different themes for you to choose from: Light, Drakula, Monkai, Solarized Light and Solarized Dark. If that does not satisfy you, ask us on our community pages and we will provide you more.</p>
-								<a class="main_btn" href="#">Get Started Now</a>
+								<h3>暗黑模式？<br />不必担心</h3>
```
Comment (ouuan) [inline content/zh/_index.html:114]:
```suggestion
								<h3>想要暗色模式？<br />我们提供了</h3>
```

File: content/zh/_index.html [hunk 13]
```diff
@@ -111,10 +111,10 @@ <h6>We know that you want to test something without a temp file</h6>
...
+								<h6>我们的生活丰富多彩，为什么不选择我们的代码编辑器呢？</h6>
```
Comment (ouuan) [inline content/zh/_index.html:115]:
```suggestion
								<h6>我们的生活是丰富多彩的，难道代码编辑器就不吗？</h6>
```

File: content/zh/_index.html [hunk 14]
```diff
@@ -111,10 +111,10 @@ <h6>We know that you want to test something without a temp file</h6>
...
+								<p>我们添加了五个不同的主题供您选择：Light, Drakula, Monkai, Solarized Light 以及 Solarized Dark。如果你不满意，请在我们的社区页面上告诉我们，我们将为你提供更多。</p>
```
Comment (ouuan) [inline content/zh/_index.html:116]:
```suggestion
								<p>我们提供了五个不同的主题供您选择：Light, Drakula, Monkai, Solarized Light 以及 Solarized Dark。如果你不满意，请告诉我们，我们将为你提供更多。</p>
```

File: content/zh/_index.html [hunk 15]
```diff
@@ -128,10 +128,10 @@ <h6>Our lives are colorful, why not our coding editor</h6>
 					<div class="row made_life_text">
 						<div class="col-lg-6">
 							<div class="left_side_text">
-								<h3>Errors and Warnings?<br />We lint them while you are coding</h3>
-								<h6>Save your precious contest time with code linting</h6>
-								<p>Using Language Servers, we provide real-time linting for errors and warnings in your code, just like Intellisense in VSCode and Atom. You can see squiggles under the warnings and hover over them to get tooltips.</p>
-								<a class="main_btn" href="#">Get Started Now</a>
+								<h3>错误和警告？<br />我们在你编程时就将它们检查出了</h3>
```
Comment (ouuan) [inline content/zh/_index.html:131]:
```suggestion
								<h3>错误和警告？<br />我们会在你打字的过程中指出</h3>
```

File: content/zh/_index.html [hunk 16]
```diff
@@ -128,10 +128,10 @@ <h6>Our lives are colorful, why not our coding editor</h6>
...
+								<h6>用代码检查节省宝贵的比赛时间 </h6>
+								<p>与 VSCode 和 Atom 中的 Intellisense 一样，使用 Language Server，我们可以为代码中的错误和警告提供实时更新。你可以在警告下方看到波浪线，并将鼠标悬停在它们上方以获取提示。</p>
```
Comment (ouuan) [inline content/zh/_index.html:133]:
```suggestion
								<p>就像 VS Code 和 Atom 中的 Intellisense 那样，使用 Language Server，我们可以为代码中的错误和警告提供实时更新。你可以在警告下方看到波浪线，并将鼠标悬停在它们上方以获取提示。</p>
```

File: content/zh/_index.html [hunk 17]
```diff
@@ -145,10 +145,10 @@ <h6>Save your precious contest time with code linting</h6>
 					<div class="row made_life_text">
 						<div class="col-lg-6">
 							<div class="left_side_text">
-								<h3>We can submit your <br/>Solutions to Codeforces</h3>
-								<h6>All thanks to Competitive Programming Community</h6>
-								<p>We have gathered the best tools in competitive programming and merged them into one editor. One such tool is called CF Tool. We have added it so you can submit and get verdicts in the editor.</p>
-								<a class="main_btn" href="#">Get Started Now</a>
+								<h3>我们可以提交你的 <br/>解决方案到 Codeforces</h3>
```
Comment (ouuan) [inline content/zh/_index.html:148]:
```suggestion
								<h3>我们可以将你的代码<br/>提交到 Codeforces</h3>
```

File: content/zh/_index.html [hunk 18]
```diff
@@ -145,10 +145,10 @@ <h6>Save your precious contest time with code linting</h6>
...
+								<h6>多亏了算法竞赛编程社区</h6>
+								<p>我们收集了算法竞赛编程中最好的工具，并将它们合并为一个编辑器。其中一种这样的工具称为 CF Tool。我们已经添加了它，因此你可以在编辑器中提交并获得判决。</p>
```
Comment (ouuan) [inline content/zh/_index.html:150]:
```suggestion
								<p>我们收集了算法竞赛编程中最好的工具，并将它们合并为一个 IDE。其中一种这样的工具称为 CF Tool。我们已经添加了它，因此，你可以在 CP Editor 中提交并获取评测结果。</p>
```

File: content/zh/_index.html [hunk 19]
```diff
@@ -180,8 +180,8 @@ <h2>Unique Screens that work perfectly</h2>
 <div class="testimonials_area p_120">
 	<div class="container">
 		<div class="main_title">
-			<h2>Feedback from Community</h2>
-			<p>Our Community support us and keep us alive. We would love to hear your feedback too!<br>See what other Competitive Programmers have to say about our editor</p>
+			<h2>来自社区的反馈</h2>
+			<p>我们的社区支持我们并保持我们的生命。我们也很想听听你的反馈！<br>看看其他算法竞赛程序员对我们的编辑器有什么看法</p>
```
Comment (ouuan) [inline content/zh/_index.html:184]:
```suggestion
			<p>我们的社区支持我们并让我们保持活力。我们也很想听听你的反馈！<br>看看其他算法竞赛选手们对 CP Editor 有什么看法</p>
```

File: content/zh/_index.html [hunk 20]
```diff
@@ -262,10 +262,10 @@ <h4>Nitish Kumar Singh</h4>
 				</div>
 				<div class="col-lg-6">
 					<div class="left_side_text">
-						<h3>Full support for<br />C/C++, Java and Python</h3>
-						<h6>Use the language you like</h6>
-						<p>We officially support three languages in the editor. You can set your own compilation commands and pass runtime arguments to your code. It really gets exciting when you have the opportunity to use different languages in different tabs with one-click options.</p>
-						<a class="main_btn" href="#">Get Started Now</a>
+						<h3>全面支持<br />C/C++, Java 和 Python</h3>
+						<h6>使用你喜欢的语言</h6>
+						<p>我们正式支持编辑器中的三种语言。你可以设置自己的编译命令，并将运行时参数传递给代码。当你有机会在带有一键式选项的不同选项卡中使用不同的语言时，这真的很令人兴奋。</p>
```
Comment (ouuan) [inline content/zh/_index.html:267]:
```suggestion
						<p>我们支持三种编程语言。你可以设置自己的编译命令，并将运行时参数传递给代码。令人兴奋的是，你有机会在带有一键式选项的不同选项卡中使用不同的语言。</p>
```

File: content/zh/_index.html [hunk 21]
```diff
@@ -278,9 +278,9 @@ <h6>Use the language you like</h6>
 <div class="impress_area p_120">
 	<div class="container">
 		<div class="impress_inner">
-			<h2>Impressed by our features?</h2>
-			<p>We are excited that you have chosen to try out this editor. <br>We hope that our Editor will fulfill all your needs in Competitive Programming. <br>CP Editor is, "by competitive programmers, to competitive programmers, for competitive programmers".</p>
-			<a class="banner_btn2" id = "download_by_os_bottom" href="https://github.com/cpeditor/cpeditor/releases/latest">Try it now</a>
+			<h2>对我们的功能印象深刻？</h2>
```
Comment (ouuan) [inline content/zh/_index.html:281]:
```suggestion
			<h2>被 CP Editor 的强大功能所吸引了？</h2>
```

File: content/zh/_index.html [hunk 22]
```diff
@@ -278,9 +278,9 @@ <h6>Use the language you like</h6>
...
+			<p>很高兴你选择尝试该编辑器 <br>我们希望我们的编辑器能够满足你在算法竞赛编程中的所有需求。 <br>CP Editor 是, "by competitive programmers, to competitive programmers, for competitive programmers".</p>
```
Comment (ouuan) [inline content/zh/_index.html:282]:
```suggestion
			<p>很高兴你选择尝试 CP Editor。<br>我们希望 CP Editor 能够满足你在算法竞赛编程中的所有需求。<br>CP Editor 由算法竞赛选手创建，给算法竞赛选手使用，为算法竞赛选手着想。</p>
```

**Review by ouuan — CHANGES_REQUESTED** [review]
Some translations are incorrect, while others sound strange. Translations should remain the main idea unchanged, but the details can be modified to be more fluent.

**Comment by ouuan** [comment]
You can use "add to batch" to apply all suggestions in a single commit.

File: content/en/_index.html [hunk 23]
```diff
@@ -23,7 +30,7 @@
 						<h2>The Missing Editor <br />for Competitive Programmers</h2>
 						<p>CP Editor is specially designed for competitive programming. It helps you focus on your algorithms and automates compilation, execution and testing of your code. It even fetches test cases for you from different platforms and submits solutions to Codeforces!</p>
 						<a class="banner_btn" href="https://github.com/cpeditor/cpeditor">View source</a>
-						<a class="banner_btn2" id="download_by_os" href="https://github.com/cpeditor/cpeditor/releases/latest">Download</a>
```
Comment (coder3101) [inline content/en/_index.html:26]: Get for { os } becomes "Get for Android" on mobile. This isn't correct.

**Review by ouuan — APPROVED** [review]

**Pull request merged**